src/error.rs

```diff
@@ -170,6 +195,10 @@ pub enum ErrorKind {
     /// Returned when a `WarpgrapherServer` fails to start.
     ServerStartupFailed(RecvError),
 
+    /// Returned by the `From` implementation for a `r2d2::Error` object.
+    /// Conversion from `r2d2::Error`.
+    R2D2Error(String),
```

Comment:
This documentation comment tells me mechanically where the error came from, but not the circumstances under which I might encounter it.  Why would Warpgrapher return such an error?

Reply:
This error has been removed. The external error is now wrapped in a specific Warpgrapher error.